docs/src/documentation/releasenotes.md

```diff
@@ -17,6 +17,35 @@ Mit Angabe der Anzahl der geplanten Wohneinheiten oder der Geschossfläche des B
 
 Die Infrastrukturabfragen aller drei Abfragearten durchlaufen einen sogenannten Abfrageprozess in welchem die verschiedenen Rollen ihre entsprechenden Einträge vornehmen. Der Abfrageprozess wird durch Statusübergänge - Auslösen über Buttons - durchlaufen.
 
+Für alle Abfragearten, ist die Datengrundlage für die Bauratendatei und Schülerpotenzialprognose geschaffen. Dabei werden Informationen zum Grund- und Mittelschulsprengel und den Vierteln aus der Verortung herausgelesen und automatisch in ISI befüllt. Eine manuelle Änderung ist möglich. Entsprechende Validierungsregeln sind umgesetzt.
+
+Für alle Abfragearten, die eine Verbindung zu Vorgängen innerhalb der e-Akte haben, ist es möglich den entsprechenden Link zum Vorgang der e-Akte anzugeben.
+
+### Berechnungen
+
+In allen Abfragearten können in den Varianten Berechnungen zu planungsursächlichen Bedarfen an Kinderbetreuungsplätzen und in den Abfragearten Bauleitplanverfahren und weitere Verfahren zu SoBoN-Bedarfen (Sozialgerechte Bodennutzung) durchgeführt werden. Dabei gilt es zu unterscheiden, dass eine planungsursächliche Berechnung für jede Abfragevariante erfolgen muss, eine SoBoN-Berechnung nur für bestimmte Varianten ausgelöst werden darf.
+
+Bei der Berechnung greift ISI auf die in ISI hinterlegten Stammdaten zurück als auch auf Information, die vorab durch die Rolle Abfrageerstellung eingegeben bzw. durch ISI ermittelt worden sind, z.B. idealtypischen Bauraten, Förderarten mit unterschiedlichen Fördermixen.
+
+Da es nicht für alle Förderarten Orientierungswerte aus der SoBoN-Grundlagenuntersuchung gibt, wurde auch eine so genannte Umlegung der Förderarten innerhalb von ISI berücksichtigt. Dadurch ist gewährleistet, dass die in ISI hinterlegten Fördermixe für die planungsursächliche und SoBoN-ursächliche Bedarfsberechnung genutzt werden können.
+
+Für den Fall, dass alleinig die Geschossfläche Wohnen und nicht die Wohneinheiten bekannt sind, erfolgt für die planungsursächliche Bedarfsberechnung automatisch eine Umrechnung der Geschossfläche Wohnen in die Wohneinheiten je Förderart.
+
+Es können Bedarfe für die Altersgruppe der Kinderkrippen- und Kindergartenkinder errechnet werden, wobei zwischen langfristigem Bedarf und Spitzenbedarf für einzelne Plätze und Gruppen unterschieden wird. Die Anzahl der Gesamteinwohner\*innen (10-Jahres-Mittel) steht als Information auch zur Verfügung.
```

Comment:
Das Backend führt keine Berechnung der Spitzenbedarfe, Summen und Mittelwerte durch. 

Backendseitig werde nur die Bedarfe über 20 Jahre ermittelt.

Die Ermittlung der Spitzenbedarfe, Summen und Mittelwerte findet in den Metabasereports statt.

Reply:
Alleinig die Sache unter dem Punkt Berechnungen mit  Anzeige der Ergebnisse in Metabase bzw. was wird wo berechnet, ist nach meinem Verständnis umformuliert, hier ggf. nochmals ein Review durch Boxi nötig.